plugins/person-extras/src/components/SkillEditor.svelte

```diff
@@ -0,0 +1,34 @@
+<!--
+Copyright © 2020, 2021 Anticrm Platform Contributors.
+
+Licensed under the Eclipse Public License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may
+obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script lang="ts">
+  import EditBox from '@anticrm/sparkling-controls/src/EditBox.svelte'
+
+  import { Skill } from '..'
+
+  export let skill: Skill
+</script>
+
+<div class="root">
+  <EditBox bind:value={skill.skill} width="100%" label="Skill" placeholder="Skill" />
```

Comment:
We need to think how to not put width="100%" by default, I think

Reply:
As far as I can see EditBox is not being used with different width so I've updated default width